CryptoBinPkg/Driver/readme.md

```diff
@@ -1,54 +1,112 @@
-# Crypto Driver
-
-VERSION: 2.0
-
-This is a pre-built version of BaseCryptLib and TlsLib built into binaries that make the services available to other
-modules via dynamic interfaces such as PPIs and protocols.
-
-This currently supports Openssl implementation of BaseCryptLib but this will be expanded in the future.
+# Shared Crypto Binaries
+
+Shared crypto consists of a set of pre-built binaries that provide cryptographic services. The services are exposed
+to other modules via dynamic interfaces such as PPIs in the PEI phase and protocols in the DXE and MM phases
+
+**This project does not directly implement crypto algorithms**. Crypto algorithm implementation is provided by an
+underlying crypto provider. The provided may change over time based on industry shifts in crypto requirements,
+enhancements for embedded crypto support in available crypto projects, and overall crypto provider quality and
+maintainership.
+
+- Currently, the OpenSSL crypto project is used.
+
+Any consumer of this binary should expect that the underlying crypto provider may change. Consumers are encouraged to
+participate in all aspects of shared crypto including code contributions and feedback.
+
+## Terminology
+
+- **Crypto Provider** - The project that provides an implementation of cryptographic algorithms used by this project
+  those crypto services to platform firmware.

```

Comment:
I think this is the wrong word here.

Reply:
It was an incomplete sentence I updated.